native/build.rs

```diff
@@ -48,7 +48,6 @@ fn compile_protobuf_files() {
             ],
             &["../proto", "../proto/pbmse/v1/"],
         )
-        .format(true)
```

Comment:
Any reason for this?

Reply:
`compile_protos` now just returns a `Result<()>`.